tests/base/test_extractors.py

```diff
@@ -222,7 +236,7 @@ def test_unintentional_synonyms_capitalized(component_builder):
 
 
 def test_spacy_ner_extractor(spacy_nlp):
-    ext = SpacyEntityExtractor()
+    ext = SpacyEntityExtractor({"dimensions": ["LOC"]})
```

Comment:
Let's leave this with no specified dimensions to make sure that the default will still parse the entities (i.e. that the default `"dimensions" = None` is being correctly passed to `filter_irrelevant_entities()`) . But perhaps initializing the extractor with the same config set up as below, so
```
    _config = RasaNLUModelConfig({"pipeline": [{"name": "ner_spacy"}]})
    ext = component_builder.create_component("ner_spacy", _config)
```

Reply:
Handled this by merging some of your later suggestions